File: src/main/java/model/Review.java
package model;

public class Review {
	
	private Integer notation;
	private String comment;
	

	public Review() {
		
	}
	
	
	public Review(Integer notation, String comment) {
		this.notation = notation;
		this.comment = comment;
	}


	public Integer getNotation() {
		return notation;
	}


	public void setNotation(Integer notation) {
		this.notation = notation;
	}


	public String getComment() {
		return comment;
	}


	public void setComment(String comment) {
		this.comment = comment;
	}
	
	

}

File: E-Shop-Angular/src/app/eshop/model/review.ts
export class Review {
  public get comment(): string | undefined {
    return this._comment;
  }
  public set comment(value: string | undefined) {
    this._comment = value;
  }
  public get notation(): number | undefined {
    return this._notation;
  }
  public set notation(value: number | undefined) {
    this._notation = value;
  }

  constructor(private _notation?: number, private _comment?: string) {}
}
